### PR TITLE
Add job for generic video tests for Windows.

### PR DESCRIPTION
**Pull request by sydvicious**
Added a job for testing the generic video API using youtube. This is just the job, it is not scheduled yet. It is windows-specific because the invocations via the shell are just too different between Windows and UNIX-like.

### CODE REVIEW

**Comment by sydvicious** [comment]
Let's withdraw this. Still figuring out how parameterized builds and the pulse monitor interact.

File: jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      <filter></filter>
+      <target>$WORKSPACE\mozharness\mozharness</target>
+      <selector class="hudson.plugins.copyartifact.WorkspaceSelector"/>
+    </hudson.plugins.copyartifact.CopyArtifact>
+    <hudson.tasks.BatchFile>
+      <command>set&#xd;
+</command>
+    </hudson.tasks.BatchFile>
+    <hudson.tasks.BatchFile>
+      <command>c:\mozilla-build\msys\bin\bash -xe -c &quot;export PATH=$MOZBUILDPATH; export MOZHARNESSHOME=$WORKSPACE\\\mozharness\\\mozharness; python ./run_media_tests.py --installer-url $INSTALLER_URL --test-timeout 300 -c config/treeherder_submission.py --log-level=debug --treeherder-url=&apos;https://treeherder.allizom.org/&apos; --treeherder-credentials=$TREEHERDER_CONFIG/treeherder-staging-credentials.json --s3-credentials=$TREEHERDER_CONFIG/s3-credentials.json; if [ $(grep --count &apos;Marionette: success&apos; logs/log_info.log) -eq &quot;0&quot; ]; then exit 1; fi&quot; </command>
```
Comment (whimboo) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: I don't see why you need mozilla-build here. Call the mozharness script directly.

Comment (sydvicious) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: @mjzffr , why did we do it this way?

Comment (mjzffr) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: @sydvicious I was following the pattern of other pf-jenkins windows jobs, My understanding is that they all use mozilla-build's bash and thus take advantage of the utilities installed in that environment.

Comment (whimboo) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: Given that mozharness encapsulates everything what it needs to run the jobs there should be no other external tool necessary from mozilla-build which we don't have installed on our machines. But if that is the case please let me know.

Comment (mjzffr) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: When I first set up the mozharness script on pf-jenkins, I ran into trouble with `unzip` not being found, so it doesn't encapsulate _everything_.

Comment (whimboo) [inline jenkins-master/jobs/mn-mse-generic-video-nightly-windows/config.xml]: Strange. Unzip shouldn't be necessary given that Python has a default package for that. If that is still the case we might want to fix that in mozharness.

**Pull request closed without merging**